File: Java-164/chapter.6/lab6.22/LabProgram.java
/*


Mad Libs are activities that have a person provide various words, which are then used to complete a short story in unexpected (and hopefully funny) ways.

Write a program that takes a string and integer as input, and outputs a sentence using those items as below. The program repeats until the input string is quit 0.

Ex: If the input is:

apples 5
shoes 2
quit 0

the output is:

Eating 5 apples a day keeps the doctor away.
Eating 2 shoes a day keeps the doctor away.
*/
import java.util.Scanner;

public class LabProgram {
    public static void doStuff(String a, int b) {    
        System.out.printf("Eating %d %s a day keeps the doctor away.\n",b,a);
    }
    public static void main(String[] args) {
        /* Type your code here. */
        Scanner scnr = new Scanner(System.in);
        String a;
        int b;
        while (true) {
        a = scnr.next();
        b = scnr.nextInt();
            if (a.contains("quit")) {
                scnr.close();
                break;
            } else {
                doStuff(a,b);
            }
        }
    }
}

File: Python-164/chapter.2/lab2.21/lab_program.py
# Instructions are imported from java 164 class.
# The syntax && logic flow might not follow exact instructions. 
'''
/*
Mad Libs are activities that have a person provide various words, which are then used to complete a short story in unexpected (and hopefully funny) ways.

Complete the program to read the needed values from input, that the existing output statement(s) can use to output a short story.

Ex: If the input is:

    Eric Chipotle 12 cars

the output is: 

    Eric went to Chipotle to buy 12 different types of cars.

*/
'''
first_name = input()
generic_location = input()
whole_number = input()
plural_noun = input()
print(f"{first_name} went to {generic_location} to buy {int(whole_number)} different types of {plural_noun}")